eventuate-local-java-cdc-connector-mysql-binlog/src/main/java/io/eventuate/local/mysql/binlog/MySqlBinaryLogClient.java

```diff
@@ -301,10 +321,64 @@ private long extractOffset(Event event) {
     return ((EventHeaderV4) event.getHeader()).getPosition();
   }
 
-  private void saveOffset(Event event) {
-    offset = extractOffset(event);
-    BinlogFileOffset binlogFileOffset = new BinlogFileOffset(binlogFilename, offset);
-    offsetStore.save(binlogFileOffset);
+
+  private void saveOffset() {
```

Comment:
All of this logic needs to be moved into the class I mentioned earlier. Too complex to have here.

Reply:
Got it, I will fix